### PR TITLE
Fix support for audio_filename as a key in manifest

### PR DESCRIPTION
**Pull request by redoctopus**
Using 'get' will fail if the default argument doesn't exist in the dict.

Signed-off-by: Jocelyn Huang <jocelynh@nvidia.com>

### CODE REVIEW

File: collections/nemo_asr/nemo_asr/parts/manifest.py [hunk 1]
```diff
@@ -52,8 +52,8 @@ def __init__(self, manifest_paths, labels, max_duration=None,
             item["tokens"] = self.tokenize_transcript(text)
 
             # support files using audio_filename
-            item['audio_filepath'] = item.get('audio_filename',
```
Comment (blisc) [inline collections/nemo_asr/nemo_asr/parts/manifest.py]: Your proposed code is almost functionally identical. When does this error occur?
It is only more robust is `audio_filepath` does not already exist in the item dict.

Comment (redoctopus) [reply collections/nemo_asr/nemo_asr/parts/manifest.py]: Wasn't the point of this segment of code was to support the case where audio_filepath doesn't exist but audio_filename does?

The current `get` fails for at least one of the datasets I've been using, which only uses `audio_filename`.

Comment (blisc) [reply collections/nemo_asr/nemo_asr/parts/manifest.py]: I feel like having `audio_filename` but not `audio_filepath` should throw an error as opposed to accepting both. Our documentation says to use `audio_filepath`. Where is there a file that has `audio_filename`?

Comment (redoctopus) [reply collections/nemo_asr/nemo_asr/parts/manifest.py]: (As a side note, the proposed change is actually a revert back to the original manifest code. The get was added recently.)

Comment (blisc) [reply collections/nemo_asr/nemo_asr/parts/manifest.py]: Let's make it one if statement: `if 'audio_filepath' not in item and 'audio_filename' in item:`

Comment (redoctopus) [reply collections/nemo_asr/nemo_asr/parts/manifest.py]: I assume we are okay with ignoring `audio_filename` if `audio_filepath` exists, then?

Comment (blisc) [reply collections/nemo_asr/nemo_asr/parts/manifest.py]: `audio_filepath` should exist. `audio_filename` should not exist. So yea, we should prioritize `audio_filepath` over  `audio_filename`

**Pull request merged**